### DISCUSSION TITLE
Release 5.2.0: Cassandra debian issue (Provision/dev/Core/Cassandra)

### DISCUSSION THREAD

**Discussion by gohilamariappan** [Sunbird-Ed/Community#546] [Bugs] [answered]
Hi Team!
@santhosh-tg 
While provisioning cassandra, we got error in the debain package. updated the version.

https://github.com/project-sunbird/sunbird-devops/pull/3845

Regards,
Gohila

**Comment by chitranshu-keshav** [comment]
thanks for the contribution @gohilamariappan. @santhosh-tg can you please do the needful and close this thread

**Reply by Krishnaj20** [reply]
@gandham-santhosh @santhosh-tg - Pls update the status of this thread. Is this taken care of?

**Comment by gandham-santhosh** [comment, marked as answer]
Yes @Krishnaj20 . Closing this thread as PR is merged
